identity-iota/src/did/doc/iota_document.rs

```diff
@@ -524,10 +516,11 @@ impl IotaDocument {
 
   /// Returns the Tangle index of the integration chain for this DID.
   ///
-  /// This is simply the tag segment of the [`IotaDID`].
+  /// This corresponds with the tag segment of the [`IotaDID`].
```

Comment:
```suggestion
  /// This is equivalent to the tag segment of the [`IotaDID`].
```

I feel like this would be a bit clearer.